ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Validators/ProjectDTOValidator.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace ExadelTimeTrackingSystem.Data.Validators
+{
+    using ExadelTimeTrackingSystem.BusinessLogic;
+    using ExadelTimeTrackingSystem.BusinessLogic.DTOs;
+    using ExadelTimeTrackingSystem.BusinessLogic.Services.Abstract;
+    using FluentValidation;
+    using static ExadelTimeTrackingSystem.BusinessLogic.Constants;
+
+    public class ProjectDTOValidator : BaseProjectDTOValidator<ProjectDTO>
+    {
+        private readonly IProjectService _projectService;
+
+        public ProjectDTOValidator(IProjectService projectService, IUserService userService)
+            : base(userService)
+        {
+            _projectService = projectService;
+            ConfigureRules();
+        }
+
+        private void ConfigureRules()
+        {
+            RuleFor(p => p.Id)
+                   .NotEmpty()
+                   .MustAsync(_projectService.ExistsAsync)
+                   .WithMessage(Validation.ID_DOES_NOT_EXIST);
```

Comment:
use Constants.Validation, it's just an usual naming, to see that it's constant var